docs: TypeScript support in Custom Applications and release `v21.8` (#2624)
commercetools/merchant-center-application-kit: website/src/content/getting-started.mdx

  --template starter
```

> If you want to start with a [TypeScript](https://www.typescriptlang.org/) project you can use the `--template=starter-typescript` option. Read more about TypeScript in the [Adding TypeScript](/development/adding-typescript) page.

Reviewer: NIT

```suggestion
If you want to start with a [TypeScript](https://www.typescriptlang.org/) project you can use the `--template=starter-typescript` flag. Read more about TypeScript in the [Adding TypeScript](/development/adding-typescript) page.
```

Reviewer 2: ```suggestion
To start with a [TypeScript](https://www.typescriptlang.org/) project, use the `--template=starter-typescript` flag. Read more about TypeScript in the [Adding TypeScript](/development/adding-typescript) page.
```
